doc/scss/install.scss

```diff
@@ -6,15 +6,50 @@
  * https://sass-lang.com/guide/
  */
 
-.install-instructions .sd-tab-set > label.sd-tab-label {
-  margin: 0;
-  text-align: center;
+.install-instructions .sd-tab-set {
+  &::before {
+    width: 5rem;
+    display: flex;
+    align-items: center;
+    font-weight: bold;
+  }
+
+  &.tabs-os::before {
+    content: "OS";
+  }
+
+  &.tabs-packager::before {
+    content: "Packager";
```

Comment:
Packager sounds a bit weird, maybe Installer instead, or "Install method" ? I know it was the wording used previously in scikit-learn 1.4 but still ...

Reply:
Package manager